### ISSUE TITLE
Need an architecture description equivalent to the Pro Architecture section

### ISSUE THREAD

**Issue by dougkm** [magento/devdocs#1774]
The is a description of the branching architecture but not of the infrastructure used.

Specifically, there is nothing a customer could use to compare against the Pro Architecture diagram here - http://devdocs.magento.com/guides/v2.2/cloud/architecture/pro-architecture.html#production-technology-stack.

Feedback on page: /guides/v2.2/cloud/basic-information/starter-architecture.html

**Comment by jeff-matthews** [comment]
@hguthrie, this may be related to an issue in our backlog ([MAGECLOUD-1659](https://magento2.atlassian.net/browse/MAGECLOUD-1659)). Please review and work with @dougkm on an update after the ece-tools 2002.0.10 release.

**Comment by meker12** [comment]
We added a table that shows the difference between Cloud Starter and Cloud Pro architectures. See [Magento Commerce Cloud architecture](https://devdocs.magento.com/guides/v2.2/cloud/architecture/cloud-architecture.html) and cleaned up the Starter architecture and workflow topics to remove duplication.  

Closing this issue. @dougkm If you have additional feedback about the updated content, we'd love to get additional feedback in a new issue.  Thanks!
